dolphinscheduler-dao/src/main/java/org/apache/dolphinscheduler/dao/mapper/TaskInstanceMapper.java

```diff
@@ -154,5 +154,7 @@ IPage<TaskInstance> queryStreamTaskInstanceListPaging(IPage<TaskInstance> page,
     List<TaskInstance> loadAllInfosNoRelease(@Param("processInstanceId") int processInstanceId,
                                              @Param("status") int status);
 
+    TaskInstance selectByCode(Long taskCode);
```

Comment:
If select by task code, it may exist multi records.
```suggestion
    List<TaskInstance> selectByCode(Long taskCode);
```

Reply:
So by id indeed, and will return one single record if we use id